code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1617,3 +1617,54 @@ GLOBAL_LIST_EMPTY(bloodmen_list)
 		H.blood_volume -= 35
 	spawn_atom_to_turf(/mob/living/simple_animal/hostile/asteroid/hivelordbrood/bloodling, owner, 3, TRUE) //think 1 in 4 is a good chance of not being targeted by fauna
 	next_expulsion = world.time + cooldown
+
+// CS Knife
+
+/obj/item/cs_knife
+	name = "M9 Bayonet"
+	desc = "A simple knife, one edge serraded and the other deadly sharp. It fills you with a compulsion to 'Rush B', whatever that means."
+	lefthand_file = 'yogstation/icons/mob/inhands/weapons/cs_knife_base_lefthand.dmi'
+	righthand_file = 'yogstation/icons/mob/inhands/weapons/cs_knife_base_righthand.dmi'
+	icon = 'yogstation/icons/obj/lavaland/artefacts.dmi'
+	icon_state = "cs_knife_base"
+	force = 10
+	damtype = BRUTE
+	sharpness = SHARP_EDGED
+	w_class = WEIGHT_CLASS_SMALL
+	hitsound = 'yogstation/sound/weapons/csknife.ogg'
+	attack_verb = list("slashed","stabbed")
+	var/fauna_backstab_bonus = 50
+	var/person_backstab_bonus = 10
+	item_flags = SLOWS_WHILE_IN_HAND
+	unique_reskin = list("Default" = "cs_knife_base",
+						"Marble Fade" = "cs_knife_marble",
+						"Ultraviolet" = "cs_knife_uv",
+						"Autotronic" = "cs_knife_auto",
+						"Doppler Saphine" = "cs_knife_ds",
+						"Doppler Ruby" = "cs_knife_dr",
+						"Doppler Emerald" = "cs_knife_de"
+						)
+
+/obj/item/cs_knife/Initialize()
```

Comment:
what this error means is you need to call . = ..() somewhere in here

Reply:
how about you call some bitches? and where?